inspirehep/modules/theme/templates/inspirehep_theme/accounts/login.html

```diff
@@ -17,95 +17,60 @@
 ## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 #}
 
-{% extends "inspirehep_theme/accounts/login_base.html" %}
-{% set title = "Sign in" %}
+{% extends config.SEARCH_UI_BASE_TEMPLATE %}
 
-{%- block form_auth_external scoped -%}{% endblock %}
-{% block nickname_popover_placement %}{% endblock%}
 
-{% block body %}
-{%- if current_user.is_guest %}
-  {{super()}}
-{%- else -%}
-<div class="col-xs-12 col-sm-10 col-md-8 col-lg-6 col-centered">
-  <div class="well">
-    <h1>{{ _('Hi') }} {{current_user.nickname}}!</h1>
-    <p style="margin-top: 10px;">{{ _('Where do you go from here?') }}</p>
-    <p style="margin-top: 30px;">
-    </p>
-    <div class="row">
-      <div width="100%" align="center">
-        <a href="/" class="btn btn-default">{{ _('Search') }}</a>
-        <a href="{{ url_for('webaccount.logout') }}" class="btn btn-default">{{ _('Sign out') }}</a>
-      </div>
-    </div>
-  </div>
-</div>
-{%- endif -%}
-{%- endblock body -%}
+{% from "invenio_accounts/_macros.html" import render_field, form_errors %}
+{% set title = "Sign in" %}
+
 
-{%- block form_header_message -%}
+{%- block body -%}
 <div class="col-centered">
   <p class="lead text-center">
     {{ _('Please sign in to suggest content to INSPIRE') }}
   </p>
 </div>
-{%- endblock form_header_message -%}
-
-{%- block form_auth_internal -%}
 <div class="col-xs-12 col-sm-10 col-md-8 col-lg-6 col-centered">
-  {%- block form_auth_internal_form -%}
   <div class="well">
     <br/>
-
-    {% set next = request.url if request.endpoint != 'webaccount.logout' and request.endpoint != 'webaccount.login' else config.CFG_SITE_SECURE_URL %}
+    {% set next = request.url if request.endpoint != 'security.logout' and request.endpoint != 'security.login' else config.CFG_SITE_SECURE_URL %}
```

Comment:
There is not one, should I add with a variable SITE_SECURE_URL = https://localhost:5000 ?